jib-core/src/test/java/com/google/cloud/tools/jib/http/ConnectionWithProxyCredentialsTest.java

```diff
@@ -1,167 +0,0 @@
-/*
- * Copyright 2018 Google LLC.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-
-package com.google.cloud.tools.jib.http;
-
-import com.google.api.client.http.apache.ApacheHttpTransport;
-import com.google.common.collect.ImmutableList;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.function.Consumer;
-import org.apache.http.auth.AuthScope;
-import org.apache.http.auth.Credentials;
-import org.apache.http.impl.client.DefaultHttpClient;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-
-/** Tests for {@link Connection} with setting proxy credentials. */
-public class ConnectionWithProxyCredentialsTest {
```

Comment:
Do we test anywhere else to make sure things still work with a proxy? (Do we even need to?)

Reply:
Yeah, we are covered. The test below shows that `ApacheHttpTransport` does read the properties and send it through the `Authorization` header.

> Note that I had previously committed a [test for this](https://github.com/GoogleContainerTools/jib/pull/1809/files#diff-7ca2549d686973d167e5c5a32b626108R108-R119) in #1809, so we can rest assured that this continues to work.

Reply:
Ah, I should read the description more, sorry.